build.libgit2.ps1

```diff
@@ -107,30 +107,30 @@ try {
     $ctest = Join-Path (Split-Path -Parent $cmake) "ctest.exe"
 
     Write-Output "Building 32-bit..."
-    Run-Command -Quiet { & remove-item build -recurse -force }
+    Run-Command -Quiet { & remove-item build -recurse -force -ErrorAction SilentlyContinue }
```

Comment:
I understand that this `-ErrorAction SilentlyContinue` might be a workaround for GitHub Actions specifically.

Instead of doing this, try to use `shell: powershell` instead of `shell: pwsh` in the build YAML. The difference is that `powershell` is a Windows PowerShell 5.0 included in Windows, and `pwsh` is PowerShell Core. As far as I remember, there is some difference in their behavior which leads to exactly this workaround when using pwsh.

Reply:
the remote item default functionality is to fail if the folder does not exist. Im supressing it since the next line is creating the folder regardless the environment on github doesnt seem to have the folder after I removed the git submodule init and update